crates/egui/src/animation_manager.rs

```diff
@@ -31,9 +31,10 @@ impl AnimationManager {
         &mut self,
         input: &InputState,
         animation_time: f32,
-        id: Id,
+        id: impl Into<Id>,
         value: bool,
     ) -> f32 {
+        let id = id.into();
```

Comment:
This will duplicate function body for every distinct type of `id`. To duplicate only body of conversion helper, you could introduce new private `_impl` function and use
```rust
fn xxx(id: impl Into<Id>) {
  self.xxx_impl(id.into(), ...)
}
```

Reply:
Yeah, this PR monomorphizes hundreds of functions for very little gain :(